shared/js/site.js

```diff
@@ -43,9 +35,6 @@ class Score {
     getTosdr() {
         let result = {}
 
-        // return if the list hasn't been built yet
-        if (!tosdrListLoaded) return result
-
         tosdrRegexList.some(tosdrSite => {
```

Comment:
Overall this makes sense and should be faster without waiting for `settings.ready()` but I would feel better about this if `tosdrRegexList` had a default value that won't error out if it hits `tosdrRegexList.some()` and its still undefined.

Reply:
(I realize its sync but just in case someone comes along and moves things around...)